### ISSUE TITLE
# Not able to download anything

### ISSUE THREAD

**Issue by Kira5659** [datawhores/OF-Scraper#485]
## Describe the bug
Every time I try to download content, i got the same error idk why

## To Reproduce

1:Go to Download content from a user”
2:Go to print a list of of my suscriptions
3:go to select the username and download the content

After that the scrip starts to getting all the information like video timeline photos everything but after that i got the issue

## Expected behavior
I expected download all the content without any problem

## Screenshots/Logs

Hint: Please make sure that the log formatting is readable, use a paste site
like
[Private Bin](https://privatebin.io/)

## Config
I haven't touched the config file at all

## System Info

- OS: Windows 11 Home
- Browser: Chrome
- Version 131.0.6778.140 (Official Build) (64-bit)
- python
- Installed via pip: 3.12.2
-

![image](https://github.com/user-attachments/assets/da6a665e-5126-468e-9e21-0f0f74bd8456)

**Comment by cjb900** [comment]
Well based on your errors you are not even using this scraper. Your errors reference [onlyfans-scraper](https://github.com/taux1c/onlyfans-scraper) which is not this scraper which is [ofscraper](https://github.com/datawhores/OF-Scraper). So no one here can help with the errors you are getting.

**Comment by Kira5659** [comment]
> Well based on your errors you are not even using this scraper. Your errors reference [onlyfans-scraper](https://github.com/taux1c/onlyfans-scraper) which is not this scraper which is [ofscraper](https://github.com/datawhores/OF-Scraper). So no one here can help with the errors you are getting.

you´re right sorry jajaja but now i´m using the correct one i´ve already installed following all the steps but i got this error 
![image](https://github.com/user-attachments/assets/3a98034a-24c4-4702-8fa2-ceea657589c3)

**Comment by rjDipcord** [comment]
> you´re right sorry jajaja but now i´m using the correct one i´ve already installed following all the steps but i got this error ![image](https://private-user-images.githubusercontent.com/85036149/395966900-3a98034a-24c4-4702-8fa2-ceea657589c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzMjYyNjEsIm5iZiI6MTczNDMyNTk2MSwicGF0aCI6Ii84NTAzNjE0OS8zOTU5NjY5MDAtM2E5ODAzNGEtMjRjNC00NzAyLThmYTItY2VlYTY1NzU4OWMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE2VDA1MTI0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4MTRhZjc2NTllODQ4YTQ3NTk0ZGJmMDUwYzYxODhhN2ZjY2M4MmYwZmQzNDVlMjk1OGI4ODk1MzNjZTllNGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._zFASgDubvak6JKgmXKe-GLuAzWDSuYcUygWu30PECg)

Also seeing this exact behavior.

**Comment by Kira5659** [comment]
> > you´re right sorry jajaja but now i´m using the correct one i´ve already installed following all the steps but i got this error ![image](https://private-user-images.githubusercontent.com/85036149/395966900-3a98034a-24c4-4702-8fa2-ceea657589c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzMjYyNjEsIm5iZiI6MTczNDMyNTk2MSwicGF0aCI6Ii84NTAzNjE0OS8zOTU5NjY5MDAtM2E5ODAzNGEtMjRjNC00NzAyLThmYTItY2VlYTY1NzU4OWMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE2VDA1MTI0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4MTRhZjc2NTllODQ4YTQ3NTk0ZGJmMDUwYzYxODhhN2ZjY2M4MmYwZmQzNDVlMjk1OGI4ODk1MzNjZTllNGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0._zFASgDubvak6JKgmXKe-GLuAzWDSuYcUygWu30PECg)
> 
> Also seeing this exact behavior.
> 
> The resolution seems to be here, in #484: [#484 (comment)](https://github.com/datawhores/OF-Scraper/issues/484#issuecomment-2543405270)

well i tried but even with doing that i got the issue

**Comment by rjDipcord** [comment]
> well i tried but even with doing that i got the issue

Is OFscraper in its own environment? Did you use pipx to install it initially?

For example if you used pip3 instead of pipx, then OFscraper isn't in its own environment, and the pipx version isn't going to natively work.

If that's the case, you need to do `pip3 uninstall ofscraper` and then `pipx install ofscraper`

Only then should you execute the workaround I mentioned at #484.

**Comment by cjb900** [comment]
> > Well based on your errors you are not even using this scraper. Your errors reference [onlyfans-scraper](https://github.com/taux1c/onlyfans-scraper) which is not this scraper which is [ofscraper](https://github.com/datawhores/OF-Scraper). So no one here can help with the errors you are getting.
> 
> you´re right sorry jajaja but now i´m using the correct one i´ve already installed following all the steps but i got this error ![image](https://private-user-images.githubusercontent.com/85036149/395966900-3a98034a-24c4-4702-8fa2-ceea657589c3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MDM5NzYsIm5iZiI6MTczNDQwMzY3NiwicGF0aCI6Ii84NTAzNjE0OS8zOTU5NjY5MDAtM2E5ODAzNGEtMjRjNC00NzAyLThmYTItY2VlYTY1NzU4OWMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDAyNDc1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5MTAyZWE0MDg0YzBiNWI3MzNmMjNiYTJlYzExZjdkY2RkMWE1OTNjYTgzMmM5YWZkODY2Njc4ZDA2NDQxNmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JJfMvMdqpzG0TTLaWRNclcmny6yW5h9My71OR2y_kbM)

if you confirm you have `aiolimiter` version 1.1.0 installed and are still getting this error try reinstalling ofscraper. It has fixed it for others with similar issues

**Comment by datawhores** [comment]
Closing because this issue is old
